### PR TITLE
Mepts 1080 mqmi cat 7 tpt data rec

### PR DESCRIPTION
**Pull request by TinoMuianga**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 1]
```diff
@@ -8580,19 +9045,83 @@ public CohortDefinition getMQ15DPatientsMarkedAsBreastfeeding() {
    *
    *
    * <ul>
-   *   <li>B4_2 - Filter all patients with a “’Ultima Profilaxia Isoniazida (Data Início)” (concept
-   *       id 6128) in Ficha Resumo (encounter type 53) during the Inclusion period as following:
+   *   <li>B4_1 - Filter all patients with “Ultima Profilaxia TPT (concept_id 23985) = INH (concept
+   *       = 656) and (Data Início)” (obs_datetime for concept id 165308 value 1256) in Ficha Resumo
+   *       (encounter type 53) during the Inclusion period as following:
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: @TinoMuianga can you change the spec to what RF12 says?

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 2]
```diff
@@ -8580,19 +9045,83 @@ public CohortDefinition getMQ15DPatientsMarkedAsBreastfeeding() {
...
+   *       <ul>
+   *         <li>
+   *             <p>Ultima Profilaxia TPT (concept_id 23985) = INH (concept = 656) and (Data
+   *             Início)” (obs_datetime for concept id 165308 value 1256) Encounter_datetime between
+   *             startDateInclusion and endDateInclusion
+   *             <p>Note: if there is more than one Ficha Clinica falling in the above criteria
+   *             consider the most recent one during the inclusion period.
+   *       </ul>
+   * </ul>
+   *
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: RF12 spec

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@TinoMuianga please have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java (hunk 1 above)
Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: @psmatsinhe It is done. The thing with the requirement document is that the specification is limited, on the description of the actual concepts being used.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java (hunk 2 above)
Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: Done

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**